dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
+
+Keep in mind that the actual BSQ you end up with will be net the proposal fee: if your request for 1000 BSQ is approved, and if the proposal fee is 2 BSQ, you will end up with 998 BSQ in your wallet.
+
+For compensation requests, please make sure you follow <<compensation#, established conventions and recommendations>>.
+
+If the proposal phase ends before you can make your proposal, simply wait and make your proposal in the next proposal phase.
+
+===== Other Details
+
+As illustrated in the image above, proposal transactions are published on the Bitcoin network and proposal data objects are broadcast on the Bisq P2P network. The Bitcoin transaction includes a hash of the proposal data object.
+
+After the proposal phase is over, there is a break of 150 blocks (about 1 day) to review and discuss proposals before voting begins.
+
+==== Blind Voting Phase
+
+_450 blocks, about 3 days_
+
+You may approve, reject, or abstain from casting a vote on proposals. It is highly recommended that you only approve or reject proposals for which you can make a well-informed decision.
+
+It is not necessary to have made a proposal to be able to vote—any stakeholder (anyone with BSQ) can participate in voting.
+
+Once you have made your selections, you must specify a stake and publish your votes.
+
+Votes cannot be changed once they are published.
+
+===== Voting Stake
+
+Your voting weight is determined by 2 factors: _reputation_ (how much BSQ you've earned), and _stake_ (how much of your owned BSQ you allot to your vote) that you specify in your vote transaction.
+
+_Reputation_ is the amount of BSQ earned by doing work for the Bisq network, independent of the amount of BSQ spent or sold. So if Alice earns 10,000 BSQ by working on Bisq, and she sells 8,000 BSQ for BTC to pay her bills, her reputation value remains 10,000 BSQ. Caveat: reputation decays over time. BSQ earned decays linearly over a period of 2 years as soon as it's earned. So the reputation value of Alice's 10,000 BSQ is 5,000 BSQ one year after she earns it, and zero two years later.
+
+Reputation cannot be specified when making a vote transaction—it's automatically calculated for the user from their past contributions.
+
+_Stake_ is the literal amount of BSQ specified for the vote transaction. This is BSQ you own that you would like to devote to your vote. The more BSQ you allot, the more weight your vote will hold. BSQ specified for stake is unspendable for the duration of the voting cycle.
+
+**Total voting weight is the sum of reputation value and specified stake.**
+
+Reputation is worth more than stake. The idea is that there is no better measure of good-will and dedication to the project than actually working on the project. The arrangement still allows people who buy into the project considerable influence, while hopefully minimizing the potential of whales with bad intentions from buying their way in and sabotaging the project.
```

Comment:
"Reputation is worth more than stake." not sure if that is really correct. Maybe "earned BSQ gives you more advantage than bought BSQ" ? or just leave it out it becomes clear with all the rest i think

Reply:
I changed it a little differently from your suggestion—see if it is ok?